File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int numOfP=0;
    int numOfY=0;
    string temp=s;
    for(int i =0; i<temp.size()+1;i++){
        if(isupper(temp[i]))
        {
            temp[i]=tolower(temp[i]);    
        }
        if(temp[i]=='p') 
            numOfP++;
        else if(temp[i]=='y')
            numOfY++;
    }
    
    if(numOfP==numOfY)
        return true;
    else 
        return false;
        
    
    
    

    
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s) {
    let numOfP = 0;
    let numOfY = 0;
    let temp = s.toLowerCase();

    for (let i = 0; i < temp.length; i++) {
        if (temp[i] === 'p') {
            numOfP++;
        } else if (temp[i] === 'y') {
            numOfY++;
        }
    }

    return numOfP === numOfY;
}

